web/client/plugins/MetadataExplorer.jsx

```diff
@@ -123,18 +121,21 @@ class MetadataExplorerComponent extends React.Component {
 
     render() {
         const panel = <Catalog zoomToLayer={this.props.zoomToLayer} searchOnStartup={this.props.searchOnStartup} active={this.props.active} {...this.props}/>;
-        const panelHeader = (<span><Glyphicon glyph="folder-open"/> <span className="metadataexplorer-panel-title"><Message msgId="catalog.title"/></span><span className="shapefile-panel-close panel-close" onClick={ toggleControl.bind(null, 'styler', null)}></span><button onClick={this.props.toggleControl} className="catalog-close close">{this.props.closeGlyph ? <Glyphicon glyph={this.props.closeGlyph} /> : <span>×</span>}</button></span>);
-        return this.props.active ? (
-            <ContainerDimensions>
-            { ({ width }) =>
-                <Dock dockStyle={this.props.dockStyle} {...this.props.dockProps} isVisible={this.props.active} size={this.props.width / width > 1 ? 1 : this.props.width / width} >
```

Comment:
this `size={this.props.width / width > 1 ? 1 : this.props.width / width}` has changed behaviour.

check it using a window with a width < 660px with both versions (your code and the older one)